src/rqt_logger_level/logger_level.py

```diff
@@ -65,6 +66,6 @@ def restore_settings(self, plugin_settings, instance_settings):
         # TODO: implement restore functionality for the current logger states
         pass
 
-    #def trigger_configuration(self):
+    # def trigger_configuration(self):
```

Comment:
Not sure what's going on with the commented code. But should the next few lines be indented?